components/InteriorLeftNavItem.js

```diff
@@ -8,8 +8,8 @@ const propTypes = {
   activeHref: PropTypes.string,
   tabIndex: PropTypes.number,
   onClick: PropTypes.func,
-  label: PropTypes.string.isRequired,
   blankTarget: PropTypes.bool,
+  children: PropTypes.node,
```

Comment:
@jlengstorf note that this library doesn't use the airbnb style guide - just eslint:recommended + prettier